src/Cli/dotnet/commands/dotnet-workload/WorkloadCommandParser.cs

```diff
@@ -77,6 +83,9 @@ private static int ProcessArgs(ParseResult parseResult)
             if (parseResult.HasOption(InfoOption) && parseResult.RootSubCommandResult() == "workload")
             {
                 ShowWorkloadsInfo(parseResult);
+                IReporter reporter = null;
+                reporter ??= Cli.Utils.Reporter.Output;
+                reporter.WriteLine("");
```

Comment:
I think this does the same thing and is simpler:

```suggestion
                Cli.Utils.Reporter.Output.WriteLine("");
```